app/controllers/microposts_controller.rb

```diff
@@ -5,27 +5,60 @@ class MicropostsController < ApplicationController
   before_action :logged_in_user, only: %i[create destroy]
   before_action :correct_user, only: :destroy
   def create
-    @micropost = current_user.microposts.build(micropost_params)
+    if params[:micropost][:micropost_id]
```

Comment:
I think just `@micropost = current_user.microposts.build(micropost_params)` is ok